packages/css/src/grid-cell/grid-cell.scss

```diff
@@ -5,41 +5,41 @@
 
 @import "../../utils/breakpoint";
 
```

Comment:
If we're turning it into a single component, maybe we want to merge the css files as well? Or at least have them both live in `/css/src/grid`?

Reply:
Ah yes I had planned that but forgot. A slight problem is the two README documents in the CSS package. Is a convention to separate their file names better than directories?

For the React components, having everything in one directory is fine. If we would get tokens for grid cell, they would live in two files. The stories are now also separate for Page Grid and Grid Cell. We can also keep the CSS components separate?

Reply:
Merged grid and grid cell into one directory in our packages, but not in Storybook. Therefore, chose to have `README.grid.md` and `README.grid-cell.md` in the CSS package as both stories reference this separately.